serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -1,26 +1,43 @@
-Working with Callbacks
-======================
+= Callback state in Serverless Workflow
```

Comment:
so, which is the expected format
a lot of ========= or just =, it will be nice to unify the criteria, because initially I use just one =, I was asked to swith to ====== and now It seems we are going back to =?

Reply:
I'd suggest using (=) heading levels, as that would help us in modularization.

Reply:
Opened #78